src/app/pages/sharing/iscsi/iscsi-wizard/iscsi-wizard.component.ts

```diff
@@ -859,4 +864,16 @@ export class IscsiWizardComponent {
             }
         }
     }
+
+    blurEvent(parent){
```

Comment:
on second thought, rather than doing this in a blur event you could do this in a validator, this would make validation actually blocking on the form and show red text

Reply:
accepted changes